tests/test_tasks.py

```diff
@@ -5104,83 +5104,3 @@ def add_em_up(*args, **kwargs):
         assert await get_background_task_run_parameters(
             add_em_up, future.state.state_details.task_parameters_id
         ) == {"parameters": {"args": (42,), "kwargs": {"y": 42}}, "context": ANY}
-
-
-def test_rollback_errors_are_logged(caplog):
```

Comment:
this entire block was just cut & pasted into test_task_engine.py::TestTransactionHooks. 
no changes other than adding `self` args.